t/command-role-location.t

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use Whatbot::Test;
+
```

Comment:
there's no test for the fail case (something without a comma or digit)
(but anyway I don't think there should be a fail case)

Reply:
Resolved with removal of the digit/comma check.